CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# v0.900.0
+This is a major overhaul that brings signficant performance improvements.
```

Comment:
```suggestion
This is a minor overhaul that brings signficant performance improvements.
```

Just to make sure that we stick with semantic versioning.

Reply:
I meant to say "major" as in "substantial". Reworded the sentence 😉